scripts/format.js

```diff
@@ -42,10 +42,13 @@ async function main() {
 
 function parseArgs() {
   return require('yargs')
-    .usage('Usage: format [commitHash] [options]')
+    .usage('Usage: format [files] [options]')
     .example('format', 'Run format only on changed files')
-    .example('format HEAD~3', 'Run format only on changed files since HEAD~3')
+    .example('format --commit HEAD~3', 'Run format only on changed files since HEAD~3')
     .options({
+      commit: {
+        description: 'Run format on files in commit only',
```

Comment:
this confusing tbh. I'd recommend to name this `--since` and update the descriptions accordingly

Reply:
I was simply aligning with the previous `commitHash` naming. I'm not entire certain on `since` since it could mean a date or a point in time ?

Reply:
`--since` is API of `lage`  that does same thing, that's why I suggest to use it here as well for consistency

`'Run format on files in commit only',` - this is not true, it will run on all commits till provided commits, not on provided commit